packages/postcss-normalize-display-values/src/__tests__/index.js

```diff
@@ -1,26 +1,40 @@
 import { test } from 'uvu';
-import mappings from '../lib/map';
-import getData from '../../../../util/getData';
 import {
   usePostCSSPlugin,
   processCSSFactory,
 } from '../../../../util/testHelpers';
 import plugin from '..';
 
 const { processCSS, passthroughCSS } = processCSSFactory(plugin);
-const data = getData(mappings);
 
 test('should pass through "block ruby"', passthroughCSS('display:block ruby;'));
 
 test('should pass through single values', passthroughCSS('display:block;'));
-
-Object.keys(data).forEach((key) => {
-  const actual = data[key];
-  const expected = key;
-
+/* source: https://www.w3.org/TR/css-display-3/#the-display-properties */
+[
```

Comment:
Use a hard coded map because if the test uses the same map as the main code, the test cannot detect mistakes in the map from the main code (for example a typo).